packages/gcf-utils/package.json

```diff
@@ -28,9 +28,12 @@
     "@types/ioredis": "^4.14.8",
     "@types/lru-cache": "^5.1.0",
     "@octokit/plugin-enterprise-compatibility": "1.2.5",
+    "@types/pino": "^6.3.0",
     "express": "^4.17.1",
     "gaxios": "^3.0.0",
+    "pino": "^6.3.2",
     "probot": "^9.11.2",
+    "sonic-boom": "^1.0.1",
```

Comment:
It looks like this could be a development dependency, as it's only used for types in tests (_pino will pull in this dep for you_).

It might be worth simply assuming you will have `sonic-boom` available, through `pino`, and simply adding an ignore rule for linting.

Reply:
Yeah the linting gave me an `extraneous dependency` error for this so I had to list it out. But since we're abstracting away Pino now, I don't think I'll need this here anyway